orgs-using-d.dd

```diff
@@ -0,0 +1,123 @@
+        )
+    )
+    $(COMMENT
+    $(DORG Remedy Games, http://remedygames.com, remedy.svg,
+        Triple-A game development,
+        First Triple-A game $(I $(HTTP quantumbreak.com, Quantum break)) with D code.
+        $(LINK_ROW
+            $(FA_TALK dconf.org/2016/talks/watson.html, DConf talk).
+        )
+    )
+    )
+    $(DORG Weka.IO, http://www.weka.io, weka.png,
+        Software defined storage,
+        Entire infrastructure and network stack (over 200K LoC).
+        $(LINK_ROW
+            $(FA_TALK dconf.org/2016/talks/zvibel.html, DConf talk) $(FA_SEPARATOR)
+            $(FA_HIRING www.weka.io)
+        )
+    )
+    $(DORG Funkwerk AG, http://www.funkwerk.com/en/, funkwerk.png,
+        Passenger information systems,
+        $(FA_ICON quote-left)
+        We use D to achieve perfect $(COMMA) well-readable code.
+        $(FA_ICON quote-right)
+        $(LINK_ROW
+            $(FA_TALK dconf.org/talks/rohe.html, DConf talk) $(FA_SEPARATOR)
+            $(FA_HIRING www.funkwerk.com/karriere/stellenangebote, hiring)
+        )
+    )
+    $(COMMENT
+    $(DORG Adroll, https://www.adroll.com, adroll.png,
+         Retargeting platform,
+         $(HTTP tech.adroll.com/blog/data-science/2015/12/08/data-science_event_processing.html, Integral parts) of $(HTTP tech.adroll.com/blog/data-science/2015/08/25/factorization-machines.html,Adroll's business) on D.
+        $(LINK_ROW
+            $(FA_TESTIMONIAL) Testimonial: $(I $(HTTP tech.adroll.com/blog/data/2014/11/17/d-is-for-data-science.html, D is for Data Science)).
+         )
+    )
+    )
+    $(DORG Economic modelling, http://www.economicmodeling.com, emsi.png,
+        Data-driven modelling,
+        $(FA_GITHUB economicmodeling)
+    )
+    $(DORG Tamedia digital, http://tamediadigital.ch/?lang=en, tamedia_digital.png,
+         Largest swiss media company,
+         $(FA_GITHUB tamediadigital)
+    )
+    $(DORG Infognition, http://www.infognition.com/, infognition.svg,
+        Video processing,
+        $(FA_TESTIMONIAL) $(HTTP www.infognition.com/blog/2014/why_d.html, Testimonial)
+    )
+
+    $(COMMENT
+
+    More links or information is needed:
+
+    $(DORG SR Labs, http://www.srlabs.it, srlabs.png,
+        Eye tracking,
+    )
+
+    Permission pending:
+
+        $(DORG eBay, https://ebay.com, ebay.svg,
+            E-commerce company. $(HTTPS github.com/eBay/tsv-utils-dlang, Github)
+        )
+
+        $(DORG RandomStorm, https://www.randomstorm.com, randomstorm.png,
+            Network security products and services.
+        )
+
+    )
+
+)
+
+$(P
+    If your organization uses D and you want it listed on this page,
+    please open a
+    $(LINK2 https://github.com/dlang/$(PROJECT)/edit/master/$(SRCFILENAME), PR) or
+    $(LINK2 https://issues.dlang.org/enter_bug.cgi?component=dlang.org&short_desc=Add%20(insert%20your%20organization)%20to%20organizations%20using%20D, issue)
+)
+
+$(T style,
+#content, #content h1 {
+    text-align: center;
+}
+)
```

Comment:
I don't like putting a `<style>` block in the document body like this. I understand it's because you can't get a hold of the `<h1>` otherwise.

The body id is supposed to help here, but we're generating an invalid one with spaces in it. However, I think you can override `BODYCLASS` here in the *.dd file:

    BODYCLASS=doc organizations-using-d

Then select by it in CSS:

    body.organizations-using-d #content h1 { text-align: center; }